docs/api-examples-source/widget.sidebar.py

```diff
@@ -0,0 +1,13 @@
+import streamlit as st
+import pandas as pd
+
+df = pd.DataFrame({
+  'first column': [1, 2, 3, 4],
+  'second column': [10, 20, 30, 40]
+})
+
+option = st.sidebar.selectbox(
+    'Which number do you like best?',
+     df['first column'])
+
+'You selected:', option
```

Comment:
We don't need this file in this case, since nothing in this PR uses the `.. output::` directive. 

More into: These files are here so we know how to regenerate the iframe'd examples in our docs. Those iframes are inserted into the docs with the directive above. For the sidebar we don't have such an iframe.

Reply:
Got it. I'll remove this one.